### PR TITLE
completed javascript-loops exercise.

### PR DESCRIPTION
**Pull request by daniel-sungwon-lee**
image

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = []
+  var currentNumber =1
+  for (currentNumber; currentNumber<11; currentNumber++) {
```
Comment (BrettAlbright) [inline javascript-loops/main.js]: This should be a while loop

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    numbers.push(currentNumber)
+  }
+  return numbers
+}
+
+function getEvenNumbersToTwenty() {
+  evenNumbers = []
+  currentNumber = 2
+  for (currentNumber; currentNumber < 21; currentNumber+=2) {
```
Comment (BrettAlbright) [inline javascript-loops/main.js]: this should also be a while loop

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates!

**Comment by daniel-sungwon-lee** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**